packages/legend-application-query/src/stores/QueryBuilderValueSpecificationBuilderHelper.ts

```diff
@@ -175,6 +180,7 @@ export const buildPropertyExpressionChain = (
               nextExpression instanceof AbstractPropertyExpression
                 ? nextExpression
                 : undefined,
+              isDatePropagationNotSupported,
```

Comment:
why do we need to pass this to `isDefaultDatePropagationSupported()` anymore? we can just stop here and do `!disableDatePropagation && isDefaultDatePropagationSupported(...)`